src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/UpgradeTarget.cs

```diff
@@ -6,6 +6,7 @@ namespace Microsoft.DotNet.UpgradeAssistant
     public enum UpgradeTarget
     {
         LTS,
-        Current
+        Current,
```

Comment:
Was surprised to see we already had this class. How does the UpgradeTarget get reflected in our integration testing?

Reply:
It doesn't - we just use the default. The one place where it actually matters has unit tests that cover it.